zrml/parimutuel/src/lib.rs

```diff
@@ -405,10 +446,27 @@ mod pallet {
 
             T::AssetManager::withdraw(winning_asset, &who, withdrawn_asset_balance)?;
 
-            let remaining_bal = T::AssetManager::free_balance(market.base_asset, &pot_account);
+            let remaining_bal =
+                T::AssetManager::free_balance(market.base_asset.into(), &pot_account);
             let base_asset_payoff = payoff.min(remaining_bal);
 
-            T::AssetManager::transfer(market.base_asset, &pot_account, &who, base_asset_payoff)?;
+            T::AssetManager::transfer(
+                market.base_asset.into(),
+                &pot_account,
+                &who,
+                base_asset_payoff,
+            )?;
+
+            if outcome_total == winning_balance {
+                let destroy_result = T::AssetDestroyer::managed_destroy(winning_asset, None);
+                unreachable_non_terminating!(
+                    destroy_result.is_ok(),
+                    LOG_TARGET,
+                    "Can't destroy winning outcome asset {:?}: {:?}",
+                    winning_asset,
+                    destroy_result.err()
+                );
```

Comment:
All the losing assets are already destroyed at this point since `on_resolution` was called before. Specifically this: https://github.com/zeitgeistpm/zeitgeist/blob/7726646fd292e8ddfb71ba91261bccea93637264/zrml/parimutuel/src/lib.rs#L632-L643

Reply:
correct thanks